packages/js-moi-providers/__tests__/jsonrpc-provider.test.ts

```diff
@@ -4,18 +4,21 @@ import { JsonRpcProvider } from "../src/jsonrpc-provider";
 import { Filter, InteractionReceipt } from "../types/jsonrpc";
 import { initializeWallet } from "./utils/utils";
 
+const HOST = "http://localhost:1600";
+const MNEMONIC = "main story burst wonder sausage spice okay pioneer person unaware force bubble";
+const ADDRESS = "0x2835c601ccd1fc6518ae17b58dc91a644d43b734001dd2cd262a51f03a3fafef";
+
 describe("Test JsonRpcProvider Query Calls", () => {
-  const address = "0x898ca25ac7a51a36894b9c9f55ec6212500dd8e0c01f6591f0eb9f5b0bc84655";
-  const mnemonic = "hockey airport rather chef nasty shrimp tragic embrace olive another own hen";
-    let provider: JsonRpcProvider;
+    const address = ADDRESS;
+    const provider = new JsonRpcProvider(HOST);
     let ixHash: string;
     let signer: Signer;
     let ixReceipt: InteractionReceipt;
     let nextNonce = 0;
+    const supply = Math.floor(Math.random() * 1000);
```

Comment:
nitpick, move the supply to the top, and place it with other constants.

Reply:
4535277a344fe6e3a959a6a82168025d98f45afa